src/client/components/filter-tile/filter-tile.tsx

```diff
@@ -251,6 +251,9 @@ export class FilterTile extends React.Component<FilterTileProps, FilterTileState
     const { essence: { filter } } = this.props;
     const dimension = DragManager.draggingDimension();
     if (dimension) return !filter.getClauseForDimension(dimension);
+    if (DragManager.isDraggingSplit()) {
+      return !filter.clauseForReference(DragManager.draggingSplit().reference);
```

Comment:
I would put all the logic in the DragManager, eg. DragManager.canDropFilterIntoSplit(filter.reference). But I'm not self-convinced.

Reply:
So we will get three functions: `canDropIntoSplit :: splits -> boolean`, `canDropIntoSeries :: series -> boolean`, `canDropIntoFilter :: filter -> boolean`. I can vouch for that ...

Reply:
Sometimes, we need to check dimension in dataCube, `DragManager` will start be aware of essence and dataCube. That seems wrong.